torch/linalg/__init__.py

```diff
@@ -140,3 +140,28 @@
     >>> LA.norm(m[0, :, :]), LA.norm(m[1, :, :])
     (tensor(3.7417), tensor(11.2250))
 """)
+
+tensorinv = _add_docstr(_linalg.linalg_tensorinv, r"""
+linalg.tensorinv(input, ind=2) -> Tensor
+
+Returns the inverse of a given tensor.
+
+This function computes a tensor such that ``tensordot(tensorinv(input), input, ind) == I``,
+where ``I`` denotes the identity tensor.
+
```

Comment:
A "Supports..." statement like that for torch.sub should appear here.

Does this support autograd?